nixos/modules/security/apparmor-suid.nix

```diff
@@ -21,29 +21,39 @@ with lib;
   };
 
   config = mkIf (cfg.confineSUIDApplications) {
-    security.apparmor.profiles = [ (pkgs.writeText "ping" ''
-      #include <tunables/global>
-      /run/wrappers/bin/ping {
-        #include <abstractions/base>
-        #include <abstractions/consoles>
-        #include <abstractions/nameservice>
+    security.apparmor.policies."bin/ping".profile = ''
+      include <tunables/global>
+      /run/wrappers/wrappers.*/ping {
+        include <abstractions/base>
+        include <abstractions/consoles>
+        include <abstractions/nameservice>
 
         capability net_raw,
         capability setuid,
         network inet raw,
 
-        ${pkgs.stdenv.cc.libc.out}/lib/*.so mr,
-        ${pkgs.libcap.lib}/lib/libcap.so* mr,
-        ${pkgs.attr.out}/lib/libattr.so* mr,
-
+        ${getLib pkgs.stdenv.cc.cc}/lib/*.so* mr,
```

Comment:
This assumes dynamic linking. What if we have static linking?

Reply:
Then this path wouldn't be relevant or exist, so the rule is harmless anyway, no?

Reply:
I don't think we have a statically linked ping.